lib/inspector.js

```diff
@@ -82,6 +86,37 @@ class Session extends EventEmitter {
   }
 }
 
+// URL construction helper functions.
+function addr() {
+  const id = wsId();
+  if (!id)
+    throw new Error('inspector did not start');
+  return wsHost() + ':' + wsPort() + '/' + id;
+}
+
+function chromeUrl() {
+  return 'chrome-devtools://devtools/bundled/inspector.html'
+       + '?experiments=true&v8only=true'
+       + '&ws=' + addr();
+}
+
+function wsUrl() {
+  return 'ws://' + addr();
+}
+
+
 module.exports = {
+  ws: {
+    id: wsId,
+    listen: wsListen,
+    unlisten: process._debugEnd,
+    // XXX find a way to just stop Io thread, not entire agent, so ID can be stable?
```

Comment:
@eugeneo Am I thinking of it wrong? Would it be useful to have a way to stop the listener thread, then be able to start it again later with the same UUID? Or does it just not matter? I'm not sure what the UUID is supposed to identify, but if its a single still-living instance of the same process, then it should be persistent (debugEnd clears it). If its supposed to identify a specific instance of an agent listening on a TCP port, and change when the port is closed, the current behaviour is fine.